lms/djangoapps/teams/static/teams/js/spec/views/team_profile_spec.js

```diff
@@ -111,7 +121,7 @@ define([
                     view = createTeamProfileView(requests, {membership: DEFAULT_MEMBERSHIP});
 
                 expect(view.$('.new-post-btn').length).toEqual(1);
-                clickLeaveTeam(requests, view);
+                clickLeaveTeam(requests, view, true);
```

Comment:
Nit: I'd prefer this be passed as an options object so that it is easier to see what the ``true`` means.

    clickLeaveTeam(requests, view, {cancel: true});